installing/installing_sno/installing-microshift-on-rhel.adoc

```diff
@@ -0,0 +1,7 @@
+:_content-type: ASSEMBLY
+[id="installing-microshift-on-rhel"]
+= Installing MicroShift on {op-system-first}
+include::_attributes/common-attributes.adoc[]
+:context: microshift-installation-overview
+
+To install MicroShift in a production-like environment, you must create a {op-system} ISO installer that includes the necessary components pre-loaded on the image. 
```

Comment:
```suggestion
To install MicroShift in a production-like environment, you will usually create a {op-system} ISO installer that includes the necessary components pre-loaded on the image. 
```

Having a system image is not an absolute requirement, but it's the most common way we expect for edge deployments.